index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="reset.css">
+        <title>Dashboard</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <script src="https://kit.fontawesome.com/d19ba74e3d.js" crossorigin="anonymous"></script>
+    </head>
+    <body>
+        <div class="header">
+            <form class="header__form">
+                <i class="fas fa-search icon"></i>
+                <input class="header__form--input" type="search" placeholder="Search"
+                    id="search">
+            </form>
+            <div class="header_user">
+                Hi, Laura Jane
+                <i class="fas fa-user-circle user-icon"></i>
+            </div>
+        </div>
+        <div class="menu">
+            <div class="menu__header">
+                YOUR LOGO
+                <i class="fas fa-bars"></i>
+            </div>
+            <div class="menu__content">
+                <div class="menu__content--link">
+                    <span>Dashboard</span>
+                    <i class="fas fa-home"></i>
+                </div>
+                <div class="menu__content--link">
+                    <span>Messages</span>
+                    <i class="fas fa-envelope"></i>
+                    <span class="badge">3</span>
+                </div>
+            </div>
+        </div>
+        <div class="main-section">
+            <h2 class="main-section__title">Overview</h2>
+            <div class="main-section__content">
+                <div class="main-section__content--left">
+                    <div class="items top-left"></div>
```

Comment:
use singular name `item`, plural names should be use only for parents not children